src/store/edit-profile/saga.test.ts

```diff
@@ -140,6 +141,27 @@ describe('updateUserProfile', () => {
   });
 });
 
+describe('fetchOwnedZIDs', () => {
+  it('fetches owned ZIDs', async () => {
+    const ownedZIDs = ['0://zid:1', '0://zid:2'];
+
+    const {
+      storeState: { editProfile },
+    } = await expectSaga(fetchOwnedZIDs)
+      .provide([
+        [
+          call(apiFetchOwnedZIDs),
+          ownedZIDs,
+        ],
+      ])
```

Comment:
FYI, if you remove the trailing commas the formatter will pull this onto a single line. It reads much better.

Reply:
got it, thanks